assets/figures/yap_system.tex

```diff
@@ -0,0 +1,60 @@
+
+
+
+\begin{tikzpicture}[scale = 0.67]
+
+\definecolor {dgray}{rgb}{122,122,122}
+\definecolor {lgray}{rgb}{194,194,194}
+
+% Payee
+	\node 			(payee) at (12,7) {\includegraphics[width = 1 cm]{../assets/images/avatar.png}};
+
+% Payer
+	\node 			(payer) at (0,7) {\includegraphics[width = 1 cm]{../assets/images/avatar.png}};
+	
+
+% Stone
+	\node (outer) at (10.5,1.5) [circle, minimum width=2.5cm, draw = gray, fill = lightgray]{};
+
+	\node (inner) at (10.5,1.5) [circle, minimum width=0.9cm, draw = gray, fill = white]{};
+
+
+% Network
+	\node			(agenta) at (1,2.8) {\includegraphics[width = 0.6 cm]{../assets/images/avatar.png}};
```

Comment:
I am indifferent between the hand-drawn agents and the stylized vector images. However, I'd prefer some consistency throughout the lecture. @golmit @dariothuerkauf please quickly talk to @kmschuler and try to reach an agreement that works for all slide decks.

Reply:
@kmschuler Please check and add my suggestions in a batch-commit, if ok.

Reply:
Replaced avatars with stickfigures in commit cd88ce5. @golmit, @dariothuerkauf, FYI.

Reply:
Note that in commit 6da3574 the old stickfigures were added to the repo. If we merge main into this branch we will be able to use the old stickfigures (where applicable) and add the new stickfigures to the collection.